app/views/admin/admin_users/confirm_delete_user_creations.html.erb

```diff
@@ -8,20 +8,20 @@
   <p class="caution notice">
     <%= ts("Are you sure you want to <strong><em>delete</em></strong> all the works and comments created by this user,
            along with their %{bookmarks} bookmarks, %{series} series, and %{collections} collections? This <strong>cannot be undone</strong>.",
-           bookmarks: @bookmarks.count, series: @series.count, collections: @collections.count).html_safe %>
+           bookmarks: @bookmarks.size, series: @series.size, collections: @collections.size).html_safe %>
   </p>
 
-  <% if @works.count > 0 %>
```

Comment:
Should these be unless @works.size.zero?

( This really is a style question, I don't know what the accepted practice is )